source/operations/install-deploy-manage/deploy-operator-kustomize.rst

```diff
@@ -0,0 +1,218 @@
+
+   For testing purposes, you can access Operator Console by configuring a NodePort using the following Kustomize patch:
+
+   .. code-block:: shell
+      :class: copyable
+
+      kubectl patch service -n minio-operator console -p '
+      {
+          "spec": {
+              "ports": [
+                  {
+                      "name": "http",
+                      "port": 9090,
+                      "protocol": "TCP",
+                      "targetPort": 9090,
+                      "nodePort": 30090
+                  },
+                  {
+                      "name": "https",
+                      "port": 9443,
+                      "protocol": "TCP",
+                      "targetPort": 9443,
+                      "nodePort": 30433
+                  }
+              ],
+              "type": "NodePort"
+          }
+      }'
+
+
+#. Verify the Operator installation
+
+   Check the contents of the specified namespace (``minio-operator``) to ensure all pods and services have started successfully.
+
+   .. code-block:: shell
+      :class: copyable
+
+      kubectl get all -n minio-operator
+
+   The response should resemble the following:
+
+   .. code-block:: shell
+
+      NAME                                  READY   STATUS    RESTARTS   AGE
+      pod/console-68d955874d-vxlzm          1/1     Running   0          25h
+      pod/minio-operator-699f797b8b-th5bk   1/1     Running   0          25h
+      pod/minio-operator-699f797b8b-nkrn9   1/1     Running   0          25h
+
+      NAME               TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)             AGE
+      service/console    ClusterIP   10.43.195.224   <none>        9090/TCP,9443/TCP   25h
+      service/operator   ClusterIP   10.43.44.204    <none>        4221/TCP            25h
+      service/sts        ClusterIP   10.43.70.4      <none>        4223/TCP            25h
+
+      NAME                             READY   UP-TO-DATE   AVAILABLE   AGE
+      deployment.apps/console          1/1     1            1           25h
+      deployment.apps/minio-operator   2/2     2            2           25h
+
+      NAME                                        DESIRED   CURRENT   READY   AGE
+      replicaset.apps/console-68d955874d          1         1         1       25h
+      replicaset.apps/minio-operator-699f797b8b   2         2         2       25h
+
+
+#. Retrieve the Operator Console JWT for login
+
+   .. code-block:: shell
+      :class: copyable
+
+      kubectl apply -f - <<EOF
+      apiVersion: v1
+      kind: Secret
+      metadata:
+        name: console-sa-secret
+        namespace: minio-operator
+        annotations:
+          kubernetes.io/service-account.name: console-sa
+      type: kubernetes.io/service-account-token
+      EOF
+      SA_TOKEN=$(kubectl -n minio-operator  get secret console-sa-secret -o jsonpath="{.data.token}" | base64 --decode)
+      echo $SA_TOKEN
+
+   
+#. Log into the MinIO Operator Console
```

Comment:
Not sure if all these different methods are really necessary. Perhaps reduce it to "according to your local policies" and NodePort.

Reply:
You could use tabs if you want to have all three available, with NodePorts selected, since that's what you used earlier in the tutorial.

That's
```
:selected:
```
for the tab to have it shown by default. It can be second in the list or whatever.

Reply:
Actually, I think tabs would be really good here.